### PR TITLE
commit to build slicerecon with .yml file and conda

### PR DESCRIPTION
**Pull request by schoonhovenrichard**
Steps to install slicerecon:
1) Download the file build_environment.yml
2) Run: conda env create --force -f build_environment.yml
3) Run: conda activate slicerecon_build
4) Clone the SliceRecon git by running: git clone https://github.com/cicwi/SliceRecon.git/
5) Run: cd SliceRecon
6) Run: mkdir build
7) Run: cd build
8) Run: CMAKE_PREFIX_PATH=$CONDA_PREFIX cmake ..
9) Run: make -j8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,18 @@ Dependencies
 If optional dependencies are not found using CMake's `find_package`, they will
 be cloned as a git submodule and built from source.
 
+To install SliceRecon follow the steps below:
```
Comment (ahendriksen) [inline README.md:22]: I would suggest using "build" instead of "install".

**Review by ahendriksen — CHANGES_REQUESTED** [review]
First of all: Thanks for contributing!

I have some suggestions for the pull request:

I think step 1 and 4 could be reversed for better usability. Once you have cloned the repository, you do not need to download `build_environment.yml` anymore. 

I think Jan-Willem will appreciate if I point you to https://chris.beams.io/posts/git-commit/
You will find that all commit messages are formatted according to the linked essay, see: https://github.com/cicwi/SliceRecon/commits/master

The code samples could be formatted as code, 

```
like this
```
That would make the formatting more in line with the rest of the README.

**Comment by adriaangraas** [comment]
@4128575 It is generally a good idea to fork the `develop` and make your changes and pull request from there. A common workflow is [git flow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow).

**Comment by jwbuurlage** [comment]
I would happily merge this PR if you clean it up, and follow the advice above. Please pull request against the develop branch. 

I’m closing this for now. You can cherry pick your changes against develop.

**Pull request closed without merging**